hexrd/instrument.py

```diff
@@ -112,6 +115,9 @@
 
 distortion_registry = distortion_pkg.Registry()
 
+multi_ims_key = 'mulitple'
+ims_classes = (ImageSeries, ProcessedImageSeries, OmegaImageSeries)
```

Comment:
I think we can delete this. See note below.

Reply:
We could -- I still need the `OmegaImageSeries` for the subsequent check though.